server/models/ReserveDates.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const ReservedDatesSchema = new Schema({
+
+    listing_id: {
+      type: Number,
+      required: true
+    },
+    bookedDates: {
+      type: Array,
+      required: true,
+    }
+
+});
+
+module.exports = ReservedDates = mongoose.model('reservedDates', ReservedDatesSchema);
```

Comment:
Just checking, is there a reason you did it this way? I think in past sprints, they set model first and then export that model's methods (like fine one, insert one, etc)